### ISSUE TITLE
[Terminal accessibility] JAWS does not speak anything against aria-live in terminal

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#142983]
CC @isidorn 

Version: 1.65.0-insider (system setup)
Commit: bb221a61d29deabd99ee9431736d04f2175cb596
Date: 2022-02-11T05:19:13.554Z
Electron: 16.0.8
Chromium: 96.0.4664.110
Node.js: 16.9.1
V8: 9.6.180.21-electron.0
OS: Windows_NT x64 10.0.19044

1. Load the latest JAWS Version 2022.2112.24.400.

2. Open VSCode.

3. Press Ctrl+` to open terminal.

4. Type `cd` enter or any other commands and see if you can hear any message from JAWS (it's silent).

**Comment by isidorn** [comment]
@meganrogge thanks a lot for fixing :clap

**Comment by rzhao271** [comment]
Hi @jooyoungseo has the issue been fixed for you on the latest Insiders?
I tried the steps above with shell integration disabled on pwsh, and I can hear when I'm typing in the command, but I can't hear the output. It could be that I'm using JAWS incorrectly, though. I tried the commands `ls` and `cd`.

**Comment by jooyoungseo** [comment]
Ctrl+Up/Down-Arrow keys now work for JAWS; however, aria-live terminal output still doesn't work for JAWS.

**Comment by jooyoungseo** [comment]
I would say, however, the removal of "document" role was helpful because now I can manually scroll through terminal output using Ctrl+U/Down arrows. The only issue is the aria-live event in terminal output is not read by JAWS while other screen readers, such as NVDA and Narrator do not have the issue. Thus, I have also reported this issue to the Vispero™, the JAWS company. Their own bug tracking number is 130037.

**Comment by meganrogge** [comment]
Thanks @jooyoungseo for clarifying. I believe you've said that this issue has been solved then, so I'm closing it.
